code/modules/telesci/bscrystal.dm

```diff
@@ -44,3 +45,30 @@
 	origin_tech = "bluespace=3;plasmatech=4"
 	blink_range = 4 // Not as good as the organic stuff!
 	points = 0 // nice try
+	refined_type = null
+
+// Polycrystals, aka stacks
+
+/obj/item/stack/sheet/bluespace_crystal
+	name = "bluespace polycrystal"
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "polycrystal"
+	desc = "A stable polycrystal, made of fused-together bluespace crystals. You could probably break one off."
+	origin_tech = "bluespace=6;minerals=3"
+	attack_verb = list("bluespace polybashed", "bluespace polybattered", "bluespace polybludgeoned", "bluespace polythrashed", "bluespace polysmashed")
+
+/obj/item/stack/sheet/bluespace_crystal/attack_self(mob/user) // to prevent the construction menu from ever happening
+	user << "<span class='warning'>You cannot crush the polycrystal in-hand, try breaking one off.</span>"
+	return
+
+/obj/item/stack/sheet/bluespace_crystal/attack_hand(mob/user)

```

Comment:
In all honesty, it would be better, and more consistent with how stacks work---and I almost guarantee that someone will want to use this as a building material, some day (like Goon's telecrystal)---where-upon it'll likely be done anyway.

Plus you woudln't have to add a ton of checks or anything, it's already all handled for you.